include/mbedtls/config.h

```diff
@@ -2018,14 +2018,16 @@
 /**
  * \def MBEDTLS_CTR_DRBG_C
  *
- * Enable the CTR_DRBG AES-256-based random generator.
+ * Enable the CTR-DRBG AES-based random generator.
```

Comment:
It's spelled `CTR_DRBG`, not `CTR-DRBG`. You don't need to fix pre-existing occurrences, but please don't introduce new occurrences of `CTR-DRBG`.